runcoms/zlogout

```diff
@@ -8,11 +8,14 @@
 # Execute code only if STDERR is bound to a TTY.
 [[ -o INTERACTIVE && -t 2 ]] && {
 
-# Print the message.
-cat <<-EOF
+# To print a message on logout:
+# uncomment all of the commented-out lines below, and
+# insert your content at <your message here>
 
-Thank you. Come again!
-  -- Dr. Apu Nahasapeemapetilon
-EOF
-
-} >&2
+# cat <<-EOF
+#
+# <your message here>
+#
+# EOF
+#
+# } >&2
```

Comment:
Just realized that I don't think this was supposed to be commented out.

Reply:
whoops, yeah, lemme fix.